### ISSUE TITLE
Spillway skipping some water sources[19c]

### ISSUE THREAD

**Issue by jupiterbjy** [ReikaKalseki/Reika_Mods_Issues#1757]
So far I didn't get hit with keyword 'Spillway' on google and github, so I guess I'm only one reporting this. Currertly using DrangonAPI/Rotarycraft/Reactorcraft 19c

![2018-01-20_19 03 36](https://user-images.githubusercontent.com/26041217/35182299-1fbc1ce2-fe15-11e7-89fe-98b30ea2a620.png)
![2018-01-20_00 22 39](https://user-images.githubusercontent.com/26041217/35182300-1fe6516a-fe15-11e7-9f39-280b8c9e25a0.png)
(removed half of floating water sources)

I was trying to fill that with Block filler, but it seems like it don't fill liquid so I tried Spillway - and result is as shown above. Each screenshots are tested on server and singleplay.

In addition to that, spillway won't sent liquid if Rotarycraft liquid/bedrock pipe is 'vertically' connected to nullifier from Thermal Expansion.
![2018-01-20_20 54 55](https://user-images.githubusercontent.com/26041217/35183135-4a91168e-fe24-11e7-8195-75be0a68bbe1.png)
Left is connected as L-shape, right is connected vertically. Vertically placed one is fed up with water and not working any more.

**Comment by ReikaKalseki** [comment]
My guess is that the water sources being "skipped" formed after some were drained; the cache is only built once, when the spillway is first placed. Normally, it tries to prevent water regeneration, but if anything causes block updates to that water - as a great many things do - it cannot prevent it.

**Comment by jupiterbjy** [comment]
I see, seems like it operate similarly with Block filler. Must be great if there was option for Block filler to ignore liquid and fill it up..

This is not pretty good idea but, you could add 'dummy water block' - which won't flow or act like lava - and replace all water sources within range of Spillway, then start draining. I'm not sure if it's even possible, but if it does, worth trying I guess.

Thanks for quick reply and hard working ;)

**Comment by ReikaKalseki** [comment]
> Must be great if there was option for Block filler to ignore liquid and fill it up..

Does it not?

**Comment by jupiterbjy** [comment]
I couldn't find option in config file to ignore liquid, maybe I need to check again;

**Comment by ReikaKalseki** [comment]
Why would it be in the config? That would not be configurable.

**Comment by jupiterbjy** [comment]
Oh, what you mean by 'Does it not?' was that Block filler do fill out liquid?
I did some test with Block filler, and I guess it think water as block, and try to avoid it.
(I'm not good at understanding idioms so, sorry in advance for possible mistake I made.)
![2018-01-25_16 07 54](https://user-images.githubusercontent.com/26041217/35375370-2383c7ca-01ea-11e8-9404-8fe125ba7d0c.png)

This is pretty off-topic, so I think it's ok to close this issue now.
Even if I have some ideas for Spillway, I'm not that intelligent to make it happen nor it correspond with it's original use - Surely it's not designed to drain sea
